frontend/src/tabs/zoom.pug

```diff
@@ -12,11 +12,18 @@
       v-bind:udate="info.untilDate",
       v-bind:avgsize="info.avgCommitSize")
 
+  .zoom__toggle-commit-message-body(v-if="info.user.commits.length")
+      button(v-on:click="toggleAllCommitMessagesBody()") show/hide all commit messages body
+
   .zoom__day(v-for="day in info.user.commits")
     template(v-if="day.insertions > 0")
       h3(v-if="filterTimeFrame === 'week'") Week of {{ day.date }}
       h3(v-else) {{ day.date }}
       ul
-        li(v-for="slice in day.commitResults")
+        li(v-for="slice in day.commitResults").commit-message.active
           a(v-bind:href="getSliceLink(slice)", target="_blank") {{ slice.messageTitle }}
+          i.fas.fa-file-alt(v-if="slice.messageBody.length > 0", onclick="toggleNext(this)", title="Click to view the commit message body")
           span &nbsp; ({{ slice.insertions }} lines)
+          pre(v-if="slice.messageBody.length > 0").body {{ slice.messageBody }}
+          p(v-else) No commit message body
```

Comment:
Not sure if its helpful to show `No commit message body`.
I think it is sufficient to show no icon `...`.